tests_login_with_email_address/test_invalid_login_details.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from playwright.sync_api import expect
+
+
+def test_blank_password_and_user(page):
+    # Go to https://login.pracuj.pl/
+    page.goto("https://login.pracuj.pl/")
+    # Click [data-test="button-login"]
+    page.locator("[data-test=\"button-login\"]").click()
+
+    email = page.locator("[data-test=\"input-email\"]")
+    expect(email).to_contain_text(" ")
+    assert page.inner_text(
+        "._1Y5MF > div:nth-of-type(1) ._1q5DA") == r"Wypełnij to pole.", f"Actual value: {page.inner_text('._1Y5MF > div:nth-of-type(1) ._1q5DA')}"
+
+    password = page.locator("[data-test=\"input-password\"]")
+    expect(password).to_contain_text(" ")
+    assert page.inner_text(
+        "._1Y5MF > div:nth-of-type(2) ._1q5DA") == r"Wypełnij to pole.", f"Actual value: {page.inner_text('._1Y5MF > div:nth-of-type(2) ._1q5DA')}"
+
+
+def test_unregistered_user_login(page):
+    # Go to https://login.pracuj.pl/
```

Comment:
Take a look at this comment and code. Both tell the same, so comments just darken the picture. General approach in Python is that code should be self explained, so you don't need to comment it. Usually, comments are for entire class/function description and for an entire block of code, not for a single line. Remove most of the comments from a code so it will be easier to read.